tester/orb.yaml

```diff
@@ -134,3 +134,47 @@ jobs:
           command: |
             cd <<parameters.cwd>>
             python -m pytest <<parameters.args>>
+
+  integration-tests:
```

Comment:
Let's call this `docker-compose`, I think, to go alongside the `pytest` job and avoid any assumptions here -- it's not necessarily the case that docker-compose is used only for integration tests, it could just as well get used for unit tests etc in the future.